reference/python/uv.lock

```diff
@@ -1614,249 +1733,69 @@ wheels = [
 
 [[package]]
 name = "langchain"
-version = "1.0.0"
-source = { editable = "../../../langchain/libs/langchain_v1" }
+version = "1.0.1"
+source = { git = "https://github.com/langchain-ai/langchain.git?subdirectory=libs%2Flangchain_v1#557fc9a81728dc889de5da0d9c8e956e2eac5d6a" }
 dependencies = [
     { name = "langchain-core" },
     { name = "langgraph" },
     { name = "pydantic" },
 ]
 
-[package.metadata]
```

Comment:
I would have only expected mostly additive lines to the `uv.lock`. Would it be possible to try to regenerate the lock? It seems like it got generated off of the prod configuration instead of the dev/normal `pyproject.toml`. Ideally, the lines changed should only reflect the adding of `langchain-azure-storage` and its dependencies.